platform/security/sts/security-sts-ldaplogin/src/main/java/ddf/ldap/ldaplogin/SslLdapLoginModule.java

```diff
@@ -475,69 +435,6 @@ public boolean login() throws LoginException {
     }
   }
 
-  protected LDAPConnectionFactory createLdapConnectionFactory(String url, Boolean startTls)
```

Comment:
moved this all into the LdapLoginConfig class which creates a connection pool and then registers it as a service. This class then manually injects the ConnectionPool. I had to do that because this class is constructed by Class.newInstance() then the initialize method is called however it was called more than once so I was getting ~70 connection pools per LDAP instead of 1.